Document/0x06b-Basic-Security-Testing.md

```diff
@@ -376,9 +374,76 @@ Save the IPA file locally with the following command:
 # itms-services -u "itms-services://?action=download-manifest&url=https://s3-ap-southeast-1.amazonaws.com/test-uat/manifest.plist" -o - > out.ipa
 ```
 
+##### Acquiring the App Binary
+
+1. From an IPA:
+
+   If you have the IPA (probably including an already decrypted app binary), unzip it and you are ready to go. The app binary is located in the main bundle directory (.app), e.g. "Payload/Telegram X.app/Telegram X". See the following subsection for details on the extraction of the property lists.
+
+    > On macOS's Finder, .app directories are opened by right-clicking them and selecting "Show Package Content". On the terminal you can just `cd` into them.
+
+2. From a Jailbroken device:
+
+    If you don't have the original IPA, then you need a jailbroken device where you will install the app (e.g. via App Store). Once installed, you need to extract the app binary from the app's bundle. This can be easily done with objection, example using Telegram:
```

Comment:
change sentence at the end: "objection, see the following example using Telegram:"

In the future we should also remove this with a DVIA or something similar and not use "real" apps like Telegram.

Reply:
I was using Telegram as it is open-source, should we avoid it anyway?

Reply:
Right, then let's keep Telegram.